tests/swift/TailorBearTest.py

```diff
@@ -24,22 +24,22 @@
 }
 """
 
-TailorBearWithoutConfig = verify_local_bear(TailorBear,
```

Comment:
Seems these changes are from 7bc134e59873137fb1b9f0feacea3bce77d2ed80. Please use git properly.

Reply:
@li-bouxuan can you kindly explain how do I get rid of this issue.

Reply:
You encountered this issue because you didn't rebase your commit properly. To solve this, revert your changes on `TailorBearTest.py` and `TSLintBearTest.py`.